starlite/connection.py

```diff
@@ -74,3 +75,32 @@ def auth(self) -> Auth:
     @property
     def query_params(self) -> Dict[str, Any]:  # type: ignore[override]
         return parse_query_params(self)
+
```

Comment:
can you push the changes in this file into main? this is something we need to have in place regardless of this PR